config/prow/cluster/build/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- test-pods_namespace.yaml
+- cpu-limit-range_limitrange.yaml
+- mem-limit-range_limitrange.yaml
+- gce-ssd_storageclass.yaml
```

Comment:
These resources are placed on both clusters. Using kustomize, we could easily share the manifests now :)

Reply:
good point 😄 
I created the overlay for these settings and renamed the folder from `build` to `base`. It would sound weird if the trusted cluster inherits settings from the build cluster. The build cluster having just a `base` configuration sounds ok from my point of view